Classes/IndexQueue/Queue.php

```diff
@@ -1028,4 +1028,18 @@ public function markItemAsFailed($item, $errorMessage = '')
             ]
         );
     }
+
+    /**
+     * Sets the timestamp of when an item last has been indexed.
+     *
+     * @param Item $item
+     */
+    public function updateIndexQueueByItem(Item $item)
```

Comment:
Hi thomas, i would propose to indicate in the name, that only the indextime is updated, otherwise somebody might think the whole items is getting updated. I would propose something like "updateIndexTimeByItem(Item $item)"